lib/portlayer/exec/container_cache.go

```diff
@@ -32,45 +35,41 @@ type containerCache struct {
 	cache map[string]*Container
 }
 
-var containers *containerCache
+var Containers *containerCache
```

Comment:
This poses a startup problem, no?  Someone is going to call `Containers.Container` and PANIC the PL because `Containers` is `nil`.